### DISCUSSION TITLE
How many traffic and request does Xray-Core can handle ? I'm getting errors

### DISCUSSION THREAD

**Discussion by AghaFarokh** [XTLS/Xray-core#2029] [Performance]
Hello everybody and I hope you are doing well

I'm running xray-core (x-ui) behind nginx and just nginx is just doing reverse proxy, sending requests to the xray-core
now I'm facing some errors on the nginx log like 
`connect() failed (111: Unknown error) while connecting to upstream, `
or
 `recv() failed (104: Unknown error) while proxying upgraded connection`
I did a lot of nginx config and tests nothing happened, somehow I think maybe it's not related to nginx and xray-core can't handle this many connections
btw this is my problem I'm getting 150MB of error.log every day and I did a lot of nginx config tuning
pls, tell me what are the maximum xray-core requests handling. and is there any way that I can fix this problem or do I need to reduce the number of users on the server? because I don't facing this problem on servers with a lower users

**Comment by AghaFarokh** [comment]
I need to add that my config is working and there is no problem in the connection, clients are connected to the server but there is a lack of quality that I think it's because of these errors
I can add logs and configuration if u need to check

**Reply by egasvegas1109** [reply]
Hi. Have you solved the problem? I'm interested in it too.

**Comment by eaglegolden** [comment]
i don't think the problem is with xray, I suggest that sniffing is turned off in the config on the server side
